pkg/config/config.go

```diff
@@ -47,6 +48,14 @@ func GetBinaryFullPath() string {
 
 // GetOktetoHome returns the path of the okteto folder
 func GetOktetoHome() string {
+	if v, ok := os.LookupEnv("OKTETO_HOME"); ok {
```

Comment:
Could we call this one `OKTETO_FOLDER` and keep the current behavior for `OKTETO_HOME` to avoid breaking changes?

Reply:
sure